### PR TITLE
docs: adding Ingress HTTP note.

### PR DESCRIPTION
**Pull request by zbwright**

### CODE REVIEW

File: Documentation/admin/ingress.md [hunk 1]
```diff
@@ -150,8 +150,16 @@ metadata:
 
 When rules are defined and the Ingress Controller is running, traffic to your Service will be routed appropriately.
 
+### HTTP header limitations
+
+Using reverse proxies that limit HTTP headers to 4K may result in errors for large group counts. If using HTTP headers for large cookie replacement, set the 
+`large-client-header-buffers` flag to `4 64k` on the Ingress Configmap `tectonic-custom-error` to increase the limit in Ingress to EG 64k.
```
Comment (joshix) [inline Documentation/admin/ingress.md]: s/EG/e.g.,/

- Consistency of K vs k for kilobytes
- The effect of this setting is the allocation of _4_ buffers of 64 kilobytes each

File: Documentation/admin/ingress.md [hunk 2]
```diff
@@ -150,8 +150,16 @@ metadata:
...
+
+For more information, see the nginx documentation on the [large-client-header-buffers flag][large-flag].
+
 
 [ingress-userguide]: https://kubernetes.io/docs/user-guide/ingress/
 [service-accounts]: onboard-service-account.md
 [controller-deployments]: https://github.com/kubernetes/ingress/tree/master/examples/deployment
 [nginx-ingress]: https://github.com/kubernetes/ingress/tree/master/controllers/nginx
+[large-flag]: http://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers.
```
Comment (joshix) [inline Documentation/admin/ingress.md]: shouldn't have a period at the end

File: Documentation/admin/ingress.md [hunk 3]
```diff
@@ -150,8 +150,16 @@ metadata:
 
 When rules are defined and the Ingress Controller is running, traffic to your Service will be routed appropriately.
 
+### HTTP header limitations
+
+Using reverse proxies that limit HTTP headers to 4K may result in errors for large group counts. If using HTTP headers for large cookie replacement, set the
```
Comment (kans) [inline Documentation/admin/ingress.md]: Perhaps something more like: 

`By default, Ingress limits HTTP headers to 8KB.  This limit may cause problems with applications that send very large headers, such as Tectonic Console when users are members of a very large number of groups. To increase the limit, set the flag...'

File: Documentation/admin/ingress.md (hunk 1 above)
Comment (zbwright) [reply Documentation/admin/ingress.md]: ah. that makes way more sense.

**Pull request merged**